nodebuilder/share/service.go

```diff
@@ -30,14 +32,14 @@ type Module interface {
 }
 
 func NewModule(lc fx.Lifecycle, bServ blockservice.BlockService, avail share.Availability) Module {
-	service := share.NewService(bServ, avail)
+	svc := service.NewShareService(bServ, avail)
```

Comment:
small nit but we usually refer to service as `serv`

Reply:
renamed for consistency.